### ISSUE TITLE
command 'remote-wsl.showLog' not found

### ISSUE THREAD

**Issue by idvorkin** [microsoft/vscode-remote-release#1462]
I'm not sure what made my WSL stop working, but I've tried re-installing, and can not get it working. What's interesting is the command ```'remote-wsl.showLog' is not found``` which makes me think the extension is not installed/registered properly. Help is much appreciated

**Version:** 

- Version: 1.38.1 (user setup)
- Commit: b37e54c98e1a74ba89e03073e5a3761284e3ffb0
- Node.js: 10.11.0
- V8: 6.9.427.31-electron.0
- OS: Windows_NT x64 10.0.18362

**Steps to Reproduce:** 

0. Re-install visual studio code after erasing the various .vscode,%APPDATA%/code directories.
1.  Launch WSL, connection fails.
2. Try viewing remote logs: 'remote-wsl.showLog' says not found. 

Does this issue occur when you try this locally?: Yes
Does this issue occur when you try this locally and all extensions are disabled?: Yes

**Looking at developer tools:** 

```
[Extension Host] debugger listening on port 29030
log.ts:161  INFO [remote-connection][ExtensionHost][624d7…][reconnect] starting reconnecting loop. You can get more information with the trace log level.
log.ts:161  INFO [remote-connection][ExtensionHost][624d7…][reconnect] waiting for 5 seconds before reconnecting...
log.ts:161  INFO [remote-connection][ExtensionHost][624d7…][reconnect] resolving connection...
log.ts:161  INFO [remote-connection][ExtensionHost][624d7…][reconnect] connecting to 127.0.0.1:9595...
log.ts:173   ERR [remote-connection][ExtensionHost][624d7…][reconnect][127.0.0.1:9595] received error control message when negotiating connection. Error:
log.ts:173   ERR Error: Connection error: Unknown reconnection token.
    at _ (remoteAgentConnection.ts:553)
    at o.onControlMessage.l (remoteAgentConnection.ts:189)
    at d.fire (event.ts:572)
    at v.fire (ipc.net.ts:449)
    at _receiveMessage (ipc.net.ts:732)
    at _socketDisposables.push._socketReader.onMessage.e (ipc.net.ts:684)
    at d.fire (event.ts:572)
    at g.acceptChunk (ipc.net.ts:235)
    at _register._socket.onData.e (ipc.net.ts:196)
    at Socket.t (ipc.net.ts:28)
    at Socket.emit (events.js:182)
    at addChunk (_stream_readable.js:283)
    at readableAddChunk (_stream_readable.js:264)
    at Socket.Readable.push (_stream_readable.js:219)
    at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94)
log.ts:173   ERR [remote-connection][ExtensionHost][624d7…][reconnect] A permanent error occurred in the reconnecting loop! Will give up now! Error:
log.ts:173   ERR Error: Connection error: Unknown reconnection token.
    at _ (remoteAgentConnection.ts:553)
    at o.onControlMessage.l (remoteAgentConnection.ts:189)
    at d.fire (event.ts:572)
    at v.fire (ipc.net.ts:449)
    at _receiveMessage (ipc.net.ts:732)
    at _socketDisposables.push._socketReader.onMessage.e (ipc.net.ts:684)
    at d.fire (event.ts:572)
    at g.acceptChunk (ipc.net.ts:235)
    at _register._socket.onData.e (ipc.net.ts:196)
    at Socket.t (ipc.net.ts:28)
    at Socket.emit (events.js:182)
    at addChunk (_stream_readable.js:283)
    at readableAddChunk (_stream_readable.js:264)
    at Socket.Readable.push (_stream_readable.js:219)
    at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94)
abstractExtensionService.ts:154 Extension host terminated unexpectedly. Code:  0  Signal:  null
_onExtensionHostCrashed @ abstractExtensionService.ts:154
notificationsAlerts.ts:38 Error: command 'remote-wsl.showLog' not found
    at Proxy._tryExecuteCommand (commandService.ts:80)
    at r.then.i (commandService.ts:73)
onDidNotificationChange @ notificationsAlerts.ts:38
ipc.net.ts:283 Cannot write message in a disposed ProtocolWriter
write @ ipc.net.ts:283
ipc.net.ts:284 u
write @ ipc.net.ts:284
```

**Comment by aeschli** [comment]
Please close all instances of VSCode in Windows and in the WSL instance (use `ps -ax` and kill to make sure no code server instances can be found).
The start VSCode again, open a remote WSL window and check the WSL log.

**Comment by idvorkin** [comment]
**TL;DR** To fix - erase WSL side vscode: E.g. ```rm ~/.vscode-server```

---
I did some more debugging, turns out I did have extensions enabled (logs below in case that helps you).   I suspected some corrupted installation, and  erased WSL side vscode e..g ```rm ~/.vscode-server```. All better now. 

---
Here's what the logs show for the run (Logs conveniently in ``` ~/.vscode-server/data/logs```): 

```
[2019-09-25 06:33:55.516] [remoteagent] [info] 

*
* Visual Studio Code Server
*
* Reminder: You may only use this software with Visual Studio family products,
* as described in the license https://aka.ms/vscode-remote/license
*


[2019-09-25 06:33:55.516] [remoteagent] [info] Extension host agent listening on 40899
[2019-09-25 06:33:55.798] [remoteagent] [info] ==> Received a management connection
[2019-09-25 06:33:55.801] [remoteagent] [info] ==> Received an extension host connection.
[2019-09-25 06:33:55.801] [remoteagent] [info] ==> Using UI language: en
[2019-09-25 06:33:55.835] [remoteagent] [info] Scanning extensions using UI language: en
```
and
```
[2019-09-25 06:31:43.601] [exthost] [info] ExtensionService#_doActivateExtension vscode.debug-auto-launch {"startup":true,"activationEvent":"*"}
[2019-09-25 06:31:43.601] [exthost] [info] ExtensionService#loadCommonJSModule file:///home/idvorkin/.vscode-server/bin/b37e54c98e1a74ba89e03073e5a3761284e3ffb0/extensions/debug-auto-launch/dist/extension
[2019-09-25 06:31:43.610] [exthost] [info] ExtensionService#_doActivateExtension vscode.emmet {"startup":true,"activationEvent":"*"}
[2019-09-25 06:31:43.610] [exthost] [info] ExtensionService#loadCommonJSModule file:///home/idvorkin/.vscode-server/bin/b37e54c98e1a74ba89e03073e5a3761284e3ffb0/extensions/emmet/dist/extension
[2019-09-25 06:31:43.660] [exthost] [info] ExtensionService#_doActivateExtension vscode.git {"startup":true,"activationEvent":"*"}
[2019-09-25 06:31:43.660] [exthost] [info] ExtensionService#loadCommonJSModule file:///home/idvorkin/.vscode-server/bin/b37e54c98e1a74ba89e03073e5a3761284e3ffb0/extensions/git/dist/main
[2019-09-25 06:31:43.749] [exthost] [info] ExtensionService#_doActivateExtension vscode.merge-conflict {"startup":true,"activationEvent":"*"}
[2019-09-25 06:31:43.749] [exthost] [info] ExtensionService#loadCommonJSModule file:///home/idvorkin/.vscode-server/bin/b37e54c98e1a74ba89e03073e5a3761284e3ffb0/extensions/merge-conflict/dist/extension
[2019-09-25 06:31:43.755] [exthost] [info] ExtensionService#_doActivateExtension ban.spellright {"startup":true,"activationEvent":"*"
```
